Html/users.html

```diff
@@ -1,174 +1,174 @@
+				<img id="ofekImgDiv"  src="../Images/ofek.png">
+				<a href='Index.html' style="margin-right: 30%;font-size: large;color: white">OfekTwitter</a>
+			</div>
+		</div>
+	</nav>
+</header>
+<div class="row">
+	<div class="col-sm-8 col-sm-offset-1">
+		<span>Filter by </span>
+		<input type="text" placeholder="user name"/>
+		<button>ok</button>
+	</div>
+
+</div>
+<div class="row">
+	<div class="col-md-7 col-md-offset-1 ">
+		<div class="row">
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+
+			</div>
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="panel panel-default userBox">
+					<div class="panel-body">
+						<img src="../Images/useravatar.png">
+						<button type="button" class="btn btn-primary">Follow</button>
+						<p>Marty lachtong</p>
+					</div>
+				</div>
+			</div>
```

Comment:
if all columns in a "row" are the exact same size, you can just define them all as "col" instead of "col-md-2"